src/dependency-manager/src/manager.ts

```diff
@@ -154,6 +155,7 @@ export default abstract class DependencyManager {
           subdomain = interpolateStringOrReject(subdomain, dep_component.context);
           // eslint-disable-next-line no-empty
         } catch { }
+        subdomain = `${subdomain}->${interface_name}`;
```

Comment:
Not a fan of this approach. We should keep subdomain unmodified because the subdomain is still `www` even if it has a path. Also this isn't guaranteed to be unique across components.

Reply:
Is traefik the only place where we run into conflicts between subdomains?

Reply:
The end of the flow where we create the traefik config is the only place I've seen this really turn into a problem, but we make assumptions in various places in the CLI and the manager that the subdomain is unique. The fact that it's used as a key turns into a problem even starting as early as [here](https://github.com/architect-team/architect-cli/pull/456/files#diff-38950282aef5cea0f96cba5414c96fc6597942377b028fddf996ad9b63649519R338) before we get into the manager

Reply:
I think its fine for the first pass to not allow conditional exposing of interface paths. If you do `-i www` it would just expose all subdomains with `www` even if they have different paths.

Reply:
If you make the traefik service name a ref of component/interface then it'd be unique. That's what we do on the api side. See `consulRef(service_node.ref, interface_name)`

Reply:
I can do that for the traefik service name. What about things like [this](https://github.com/architect-team/architect-cli/pull/456/files#diff-61da828261689d3f251ab9c24859dadff3bb043a5b45e54093c6f944cc045943R172) though? What should happen in a case like this if two subdomains are the same?